selfdrive/car/hyundai/values.py

```diff
@@ -131,6 +132,7 @@ class HyundaiCarInfo(CarInfo):
   ],
   CAR.VELOSTER: HyundaiCarInfo("Hyundai Veloster 2019-20", min_enable_speed=5. * CV.MPH_TO_MS, harness=Harness.hyundai_e),
   CAR.SONATA_HYBRID: HyundaiCarInfo("Hyundai Sonata Hybrid 2020-22", "All", harness=Harness.hyundai_a),
+  CAR.IONIQ_5: HyundaiCarInfo("Hyundai Ioniq 5 2022", "All", harness=Harness.hyundai_k),
```

Comment:
is the harness modified or stock k works?

Reply:
can you make this Harness.none for now?

Reply:
> is the harness modified or stock k works? @vanillagorillaa looks like we'll need another hyundai harness

What a joy, another one!